lib/api.js

```diff
@@ -307,13 +308,13 @@ function getRepos(token, username, callback) {
     },
     function fetchOrgTeams(err, r, o) {
       if (err) {
-        console.error("get_github_repos() - Error fetching repos & orgs: %s", err);
-        return callback(err)
+        console.error('get_github_repos() - Error fetching repos & orgs: %s', err);
+        return callback(err);
       }
-      if (!r) return callback('Get repos failed; no response')
+      if (!r) return callback('Get repos failed; no response');
 
       org_memberships = o.data;
-      for (var i=0; i<r.data.length; i++) {
+      for (var i = 0; i < r.data.length; i++) {
```

Comment:
I'll take a look at it and update the PR. I'll probably make another pass through the whole code anyway and look for small improvements to make.